deployment/ansible/group_vars/integration/config.yml

```diff
@@ -0,0 +1 @@
+log_level: debug
```

Comment:
I suggest warning here too - as UAT - in order not to experience only in UAT that the warning+ logs are not enough.
dev1 and dev2 ok debug.

Reply:
It sounds better to me that integration is in debug mode and UAT is not. While the integration contains unstable software, the UAT should contain stable releases only and debug shouldn't be needed except in extreme situations.

Reply:
ok